crates/matrix-sdk-crypto/src/types/events/utd_cause.rs

```diff
@@ -49,14 +58,16 @@ pub enum UtdCause {
     UnknownDevice = 4,
 
     /// We are missing the keys for this event, but it is a "device-historical"
-    /// message and no backup is accessible or usable.
+    /// message and key storage backup is disabled.
```

Comment:
The term "disabled" seems a bit handwavey. Could we be more explicit about what the potential causes are?

(explicitly disabled by user, no backup on the server (?) missing signature on the backup, we don't have the keys despite being verified (?), etc)

Reply:
I'm fairly sure we just mean "turned off by the user" here. I'll write that.

Reply:
Hmm. My logic is "does not exist on the server".